### DISCUSSION TITLE
使用谷歌拓展时在Aria2状态那边关闭Aria2结果一直连不上，点启动也没有用

### DISCUSSION THREAD

**Discussion by ajian357** [alexhua/Aria2-for-chrome#173] [Q&A]
使用谷歌拓展时在Aria2状态那边关闭Aria2结果一直连不上，点启动也没有用
![_J OFAJ67K64CA_$ X%`S5M](https://github.com/alexhua/Aria2-Explorer/assets/76824756/2a4e2285-f75b-4813-938f-23f9d0adcffe)

**Comment by ajian357** [comment]
有谁知道为什么不，网上搜不到原因，麻烦救一救

**Comment by ajian357** [comment]
> 使用谷歌拓展时在Aria2状态那边关闭Aria2结果一直连不上，点启动也没有用 ![J OFAJ67K64CA$ X%`S5M](https://private-user-images.githubusercontent.com/76824756/285595976-2a4e2285-f75b-4813-938f-23f9d0adcffe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDA5MjI3NjcsIm5iZiI6MTcwMDkyMjQ2NywicGF0aCI6Ii83NjgyNDc1Ni8yODU1OTU5NzYtMmE0ZTIyODUtZjc1Yi00ODEzLTkzOGYtMjNmOWQwYWRjZmZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI1VDE0Mjc0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUxOTQzYTJiOTliNzUzNzk4ZjkzZTY3M2NiZjRjNTI4MzM4ODk5NmVkMmU3NGEyZDc1MWIyNmVmMTFjZDhjZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EJ2EsbwGXLBhq1jM0o-Dg3PsQp2e1uEiVnvADw2YPRs)

![image](https://github.com/alexhua/Aria2-Explorer/assets/76824756/d37c0ab6-08f7-43cf-aa50-7389c603f734)
和端口有关系不

**Comment by alexhua** [comment]
![image](https://github.com/alexhua/Aria2-Explorer/assets/1220458/36462f6a-20ee-4bd0-9e6c-0f0402a26901)
这个是你吧？你这操作真是让人啼笑皆非，点了关闭肯定连不上了啊。你的Aria2服务端一开始是怎么启动的？把问题的来龙去脉和背景交代清楚别人才能帮到你。
